tuneme/tests/test_views.py

```diff
@@ -47,3 +48,37 @@ def test_report_response(self):
             response,
             "This comment has been reported."
         )
+
+    def test_commenting_closed(self):
+        client = Client()
+        client.login(username='tester', password='tester')
+        article = ArticlePage.objects.create(
+            title='article 1', depth=1,
+            subtitle='article 1 subtitle',
+            slug='article-1', path=[1], commenting_state='C')
+        article.save()
+        data = MoloCommentForm(self.user, {}).generate_security_data()
+        data.update({
+            'comment': "This is another comment",
+            'object_pk': article.id,
+            'content_type': type(article)
```

Comment:
This line looks suspicious because `type(article)` is likely the class `Article` which seems like a strange thing to try pass as a URL parameter and probably results in the strange `<class molo...` in the test failure output.